content/md/first-steps/manage-your-measurements.md

```diff
@@ -0,0 +1,141 @@
+
+![List of measurement families](../img/Settings_Measurement_Families.png)
+
+You can search on the label or code of measurement families and you can sort on all the columns.
+
+# How to create a measurement family?
+
+To create a new measurement family, follow these steps:
+1.  Go to `Settings/Measurements`
+1.  Click on `Create`
+![Create a measurement family](../img/Settings_Measurement_Families_Create.png)
+1.  The pop-in to create a measurement family is displayed
+1.  Input the new measurement family code (this code is required and must be unique)
+1.  Input its label in the current [UI locale](manage-your-account.html#your-interface-default-language)
+1.  For the standard unit used to convert the other units of this measurement family, input its code (this code is required and must be unique), its label in the current [UI locale](manage-your-account.html#your-interface-default-language) and symbol
+1.  Click on `Save`
+
+The measurement family is now created and the screen to display and [add others units](#manage-units-in-a-measurement-family) opens.
+
+::: warning
+The **standard unit** is set once and for all.
+:::
+
+:::info
+To guarantee that the PIM is functional and runs smoothly, we defined a volume limitation of 100 measurement families. If you try to create more than 100 measurement families, you will have a blocking message.  
+:::
+
+# Edit properties of a measurement family
+To edit a measurement family:
+1.  Go to `Settings/Measurements`
+2.  Select the measurement family to edit in the grid and click on the line or on the edition shortcut
+3.  Translate the labels in the `Properties` tab and click on `Save`
+
+# Manage units in a measurement family
+To manage units in a measurement family:
+1.  Go to `Settings/Measurements`
+2.  Select the measurement family to edit in the grid and click on the line or on the edition shortcut
+3.  Manage the units in the `Units` tab and click on `Save`
+
+## Add a unit to a measurement family
+Once you are on your measurement family edit form, go under the `Units` tab:
+1.  Click on `Add unit`
+1.  The pop-in to add a new unit to a measurement family is displayed
+1.  Input the new unit code (this code is required and must be unique)
+1.  Input its label in the current [UI locale](manage-your-account.html#your-interface-default-language) and its symbol
+1.  Input the conversion operations to convert the unit into the standard unit (click on `Add new operation` to add several conversion operations, click on the cross to remove an operation)
+1.  Click on `Add`, the screen with all the units opens
+1.  Translate the labels of the added unit in the `Label translations` section
+1.  Click on `Save`
+
+The unit is now added.
+
+::: warning
+When a [product attribute](manage-your-attributes.html#create-an-attribute) is linked to a measurement family, once a unit is created, you will only be able to edit its translated labels and symbol.
```

Comment:
```suggestion
If a [product attribute](manage-your-attributes.html#create-an-attribute) is linked to a measurement family, once a unit is created, you will only be able to edit its translated labels and symbols.
```

Reply:
Only one symbol and several labels per locale